src/PenSkin.js

```diff
@@ -597,6 +603,9 @@ class PenSkin extends Skin {
         gl.clearColor(0, 0, 0, 0);
         gl.clear(gl.COLOR_BUFFER_BIT);
 
+        this._silhouettePixels = new Uint8Array(Math.floor(width * height * 4));
```

Comment:
Nit: the `math.floor` isn't necessary here

Reply:
It seems to be used in every other place in the codebase that constructs a `Uint8Array`, [specifically to prevent issues with Edge](https://github.com/LLK/scratch-render/pull/132).

If I removed it here, I don't think it would cause any trouble (`width` and `height` *should* be integers), and newer versions of Edge (I tested it on Microsoft Edge 44.18362.449.0, Microsoft EdgeHTML 18.18363) have no trouble with floating-point values in typed array constructors, but it'd break the convention that the rest of the codebase is using. Is that okay?

(P.S. it may be worth revisiting this in the future by either testing whether Edge 15, the minimum version [Scratch officially supports](https://scratch.mit.edu/info/faq), supports floating-point values in `TypedArray` constructors, or by bumping the minimum supported Edge version in the FAQ to something newer.)

Reply:
Oops - thanks for the reminder! I had completely forgotten about that.